addon/components/base/bs-accordion.js

```diff
@@ -12,18 +12,18 @@ import listenTo from 'ember-bootstrap/utils/listen-to-cp';
  components as children:
 
  ```handlebars
-  {{#bs-accordion as |acc|}}
-      {{#acc.item value=1 title="First item"}}
-        <p>Lorem ipsum...</p>
-        <button {{action acc.change 2}}>Next</button>
-      {{/acc.item}}
-      {{#acc.item value=2 title="Second item"}}
-        <p>Lorem ipsum...</p>
-      {{/acc.item}}
-      {{#acc.item value=3 title="Third item"}}
-        <p>Lorem ipsum...</p>
-      {{/acc.item}}
-  {{/bs-accordion}}
+  <BsAccordion as |Acc|>
+    <Acc.item @value=1 @title="First item">
```

Comment:
Every value that isn't a constant string must be wrapped in curlies isn't it?

> If the argument value is a constant string, it can appear verbatim after the equal sign, i.e. <FooBar @foo="some constant string" />. Other values should be enclosed in curlies, i.e. <FooBar @foo={{123}} @bar={{this.bar}} />.
https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md#arguments